input/cheat-sheet.yaml

```diff
@@ -1,531 +1,645 @@
 - title:
     en: <i>A VisiData Cheat Sheet</i>
     es: <i>Un "Cheat Sheet" Para VisiData</i>
+    it: <i>Un "Cheat Sheet" per VisiData</i>
   commands:
   - cmd:
       en: Version
       es: Versión
+      it: Versione
     desc:
       en: VisiData v1.5.2
   - cmd:
       en: Updated
       es: Actualizado
+      it: Aggiornato
     desc:
       en: '2019-07-07'
       es: '2019-07-07'
+      it: '2019-07-07'
   - cmd:
       en: More info
       es: Más info
+      it: Maggiori informazioni
```

Comment:
@aborruso Would "Maggiori info" be understood? (Just to keep things *un po 'corto*.)